control-plane/pkg/reconciler/consumer/consumer.go

```diff
@@ -269,12 +269,12 @@ func (r *Reconciler) reconcileReplyStrategy(ctx context.Context, c *kafkainterna
 		return nil
 	}
 	if c.Spec.Reply.URLReply != nil && c.Spec.Reply.URLReply.Enabled {
-		destination, err := r.Resolver.URIFromDestinationV1(ctx, c.Spec.Reply.URLReply.Destination, c)
+		destination, err := r.Resolver.AddressableFromDestinationV1(ctx, c.Spec.Reply.URLReply.Destination, c)
 		if err != nil {
 			return fmt.Errorf("failed to resolve reply destination: %w", err)
 		}
 		egress.ReplyStrategy = &contract.Egress_ReplyUrl{
-			ReplyUrl: destination.String(),
+			ReplyUrl: destination.URL.String(),
```

Comment:
You have to make this change in a few other places as well

https://github.com/knative-sandbox/eventing-kafka-broker/pull/3114/files#diff-6ee4865953a537188757b144b2c85c1d9fb9cf0b206c660216e53289f2e98f2fR155
https://github.com/knative-sandbox/eventing-kafka-broker/pull/3114/files#diff-15d344ea3208a104992f6e8ee1431a4825c26171d1038fac81fddeaa5390bef1R320